### DISCUSSION TITLE
2024-08-01-Obfuscating-spatial-data/

### DISCUSSION THREAD

**Discussion by giscus** [n8layman/n8layman.github.io#9] [Comments]
# 2024-08-01-Obfuscating-spatial-data/

How can we protect sensitive information while still ensuring reproducibility?

https://n8layman.github.io/2024-08-01-Obfuscating-spatial-data/

**Comment by n8layman** [comment]
There are a lot more considerations to spatial obfuscation such as identifying how much to jitter data points and applying these ideas to raster data. For example adding 1000 ft of noise to data from someone living in a densely populated area like Manhattan may be fine and yet completely inappropriate for someone living alone on a remote mountain. More advances spatial cloaking techniques can take advantage of the spatial arrangement of the data to ensure anonymity.

**Comment by n8layman** [comment]
Inspiration from this came from a function in the [ohcleandat](https://github.com/ecohealthalliance/ohcleandat) repo.
